generators/service-appid/templates/node/README.md

```diff
@@ -0,0 +1,105 @@
+# AppID
+ 
+ 
+ App ID is an authentication and profiles service that makes it easy for developers to add authentication to their mobile and web apps, and secure access to cloud native apps and services on Bluemix. It also helps manage end-user data that developers can use to build personalized app experiences.
+##  Credentials
+
+###  LocalDevConfig
+
+This is where your local configuration is stored for AppID.
+```
+{
+  "appid_tenant_id": "{{tenantId}}", // tenant ID
+  "appid_client_id": "{{clientId}}", // client ID
+  "appid_secret": "{{secret}}", // secret
+  "appid_oauth_server_url": "{{oauthServerUrl}}", // Oauth Server Url
+  "appid_profiles_url": "{{profilesUrl}}" // Profile URL
+}
+```
+
+## Usages
+
+```javascript
+    var session = require('express-session');
+    var serviceManager = require('./services/service-manager');
+    /**
+        Note: You will need to see the appid-redirect-ui property prior to requring the service module.
+        This will ensure that your redirect url will be correct prior to initalizing the AppID SDK.
+    */
+    serviceManager.set('appid-redirect-uri', 'http://localhost:3000' + CALLBACK_URL);
+   
+    require('./services/index')(app);
+    
+    const CALLBACK_URL = "/ibm/bluemix/appid/callback";
+    const LOGIN_URL = "/ibm/bluemix/appid/login";
+
+    // Setup express application to use express-session middleware. Must be configured with proper session 
+    // storage for production environments. See https://github.com/expressjs/session for additional
+    // documentation
+    app.use(session({
+        secret: "ninpocho",
```

Comment:
Is that a working secret?  I know this is only documentation.  Maybe show just ***** with a comment that it's masked?  That at least encourages an attitude of privacy.

Reply:
Sure. It is just a random text but I can change it